site/specs/numbers.json

```diff
@@ -850,22 +847,24 @@
           "/accounts"
         ],
         "description": "<p>Retrieves the phone numbers according to the input parameters</p>\n",
-        "operationId": "GET /accounts/{accountId}/availableNumbers",
+        "operationId": "GetAvailableNumbers",
+        "summary": "Retrieves available tns",
```

Comment:
`summary` is used as the operation name in our docs, so it should be `Get available tns` or `Retrieve available tns`